ProjectStructureAndRoles.md

```diff
@@ -2,42 +2,95 @@
 
 ## Project Structure
 
+**Overview**
+
+The work of CAMARA is organized in 
+* Sub Projects and their API Repositories
+* Working Groups and their Repositories
+
+API Repositories in CAMARA are classified with one of three stages of maturity:
+
+* Sandboxed
+* Incubated
+* Graduated
+
+Sandbox API Repositories can initially exist independent of a Sub Project. The process for the creation, incubation, graduation and (potential) archiving of API Repositories is described within [API-Onboarding-and-Lifecycle.md](./API-Onboarding-and-Lifecycle.md).
+
 **Sub Projects**
 
-The CAMARA Project is organized primarily into Sub Projects. Each Sub Project is composed of Project participants from multiple companies and organizations, with a common purpose of advancing the Sub Project with respect to a specific Service API topic, for example ‘quality on demand’ or ‘localization’. Our goal is to enable distributed decision making and code ownership. This will be done by providing focused forums for getting work done, making decisions, and onboarding new Contributors.
+Sub Projects are the primary structure of the CAMARA Project. Each Sub Project is composed of Project participants from multiple companies and organizations, with a common purpose of advancing the Sub Project with respect to a specific Service API topic, for example ‘quality on demand’ or ‘localization’. Our goal is to enable distributed decision making and code ownership. This will be done by providing focused forums for getting work done, making decisions, and onboarding new Contributors.
```

Comment:
Should we include here the term family? As it has evolved from its creation, right now families are mostly parallelizing the subgroup concept, a group of APIs with a relation among them but separated in different repositories to ensure that there is not cross-dependancy in the specification. Location, qod... are good examples for API families.

Reply:
I'm a bit skeptical to re introduce the family term. We had discussion on the past but is was difficult to define and we get ride of it. Sub project term is just go enough for me.

Reply:
I'm here with Ludovic, we shouldn't use the term as an official term going forward as we have the structure elements of API Repository and Sub Projects with the "common purpose of advancing the Sub Project with respect to a specific Service API topic". We will get in addition the API Portfolio categories to structure the overall set of APIs within the communication towards API Consumers.